### ISSUE TITLE
[Vulnerabilities]Dependabot alerts: Segmentation fault in time 

### ISSUE THREAD

**Issue by p5150j** [LibertyDSNP/frequency#863]
See:
[Segmentation fault in time ](https://github.com/LibertyDSNP/frequency/security/dependabot/19)

**Comment by rlaferla** [comment]
As the `time` crate is not a direct dependency, I will revisit this after the Polkadot 0.9.36 upgrade.

**Comment by rlaferla** [comment]
We are running version 0.1.45.  The advisory states:  "Users of time 0.1 do not have a patch and must upgrade to an unaffected version: time 0.2.23 or greater or the 0.3 series."

```
[package]]
name = "time"
version = "0.1.45"
source = "registry+https://github.com/rust-lang/crates.io-index"
checksum = "1b797afad3f312d1c66a56d11d0316f916356d11bd158fbc6ca6389ff6bf805a"
dependencies = [
 "libc",
 "wasi 0.10.0+wasi-snapshot-preview1",
 "winapi",
]
```

**Comment by rlaferla** [comment]
However, the `time` crate is not a direct dependency of Frequency.  It's a dependency of `chrono` which has not updated the `time` dependency nor as per this issue will it fix the security issue:

https://github.com/chronotope/chrono/issues/602

**Comment by wilwade** [comment]
Still blocked.

**Comment by wilwade** [comment]
Fixed in Polkadot sdk 1.0
